datemath/helpers.py

```diff
@@ -175,14 +181,14 @@ def parseTime(timestamp: str, timezone: str = 'UTC') -> Arrow:
         if debug: print('parseTime() - Doesnt look like we have a valid timestamp, raise an exception.  timestamp={}'.format(timestamp))
         raise DateMathException('Valid length timestamp not provide, you gave me a timestamp of "{}", but I need something that has a len() >= 4'.format(timestamp))
           
-def roundDate(now: Any, unit: str, tz: str = 'UTC', roundDown: bool = True) -> Arrow:
+def roundDate(now: Arrow, unit: str, tz: str = 'UTC', roundDown: bool = True) -> Arrow:
     '''
         rounds our date object
     '''
     if roundDown:
-        now = now.floor(unit)
+        now = now.floor(unit)  # type: ignore[arg-type]
     else:
-        now = now.ceil(unit)
+        now = now.ceil(unit) # type: ignore[arg-type]
```

Comment:
`arrow` types these only with the literal strings they accept. I figured you'd probably prefer keeping the types and checks simpler with a more general `str`, as any usage of `roundDate` would also have to comply to only using statically known literals.